tests/helpers.py

```diff
@@ -237,6 +245,10 @@ def strip_all_whitespace(content):
         pattern = re.compile(r'\s+')
         return re.sub(pattern, '', content)
 
+    @staticmethod
+    def find_search_summary(res_data):
+        return re.findall(r'<span class="search-summary-count">.+</span>[^\n]+', res_data)
```

Comment:
Might be nicer to read into a document and find with XPath rather than a regex.

Reply:
As you noticed, this is just moving it out to a more central place.